CMakeLists.txt

```diff
@@ -84,19 +91,67 @@ if ( ament_cmake_FOUND )
     include(cmake/ament_build.cmake)
 else()
     message(STATUS "------------------------------------------")
-    message(STATUS "BehaviorTree is being built with conan.")
+    message(STATUS "BehaviorTree is being built without AMENT.")
```

Comment:
I've changed this message to better reflect what's going on. This code path doesn't necesarily means that conan is being used, it just means that ament is not used. On the docs it says that one can build without conan or ament if ZeroMQ and Sqlite3 are provided somehow (e.g. apt on ubuntu). If you do that, then you fall on this code path and the log wrongly says that conan is being used.